tree/treeplayer/inc/ROOT/TDFInterface.hxx

```diff
@@ -751,8 +751,10 @@ public:
    ////////////////////////////////////////////////////////////////////////////
    /// \brief Creates a node that filters entries based on range: [begin, end)
    /// \param[in] begin Initial entry number considered for this range.
-   /// \param[in] end Final entry number (excluded) considered for this range. 0 means that the range goes until the end of the dataset.
-   /// \param[in] stride Process one entry of the [begin, end) range every `stride` entries. Must be strictly greater than 0.
+   /// \param[in] end Final entry number (excluded) considered for this range. 0 means that the range goes until the end
+   /// of the dataset.
```

Comment:
don't formatting changes like this break the doxygen doc?

Reply:
arg. I thought we protected that. Thanks for spotting this.